ci-operator/step-registry/storage/conf/csi-openstack-cinder/storage-conf-csi-openstack-cinder-commands.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+cd /go/src/github.com/openshift/openstack-cinder-csi-driver-operator
+cp test/e2e/manifest.yaml ${SHARED_DIR}/${TEST_CSI_DRIVER_MANIFEST}
+
+# For debugging
+echo "Using ${SHARED_DIR}/${TEST_CSI_DRIVER_MANIFEST}:"
+cat ${SHARED_DIR}/${TEST_CSI_DRIVER_MANIFEST}
```

Comment:
Will this contain any secrets?

Reply:
Looks like no. This is the manifest file:
https://github.com/openshift/openstack-cinder-csi-driver-operator/blob/master/test/e2e/manifest.yaml